logger/log.go

```diff
@@ -0,0 +1,33 @@
+package logger
```

Comment:
in the Description, reference the PRs/issues about seelog when talking about why we made this change.